docs/deployment_kubernetes_docker.md

```diff
@@ -0,0 +1,199 @@
+---
+title: "Deploying on Amazon SageMaker with BentoML"
+sidebar: home_sidebar
+---
+
+
+# Kubernetes Deployment
+
+In this guide, you will deploy the pet classification model from lesson one as an API
+server to Kubernetes using BentoML.
+
+## Setup
+
+1. A Kubernetes enabled cluster or machine.
+    * learn more about kubernetes installation: https://kubernetes.io/docs/setup/
+    * This guide uses Kubernetes' recommend learning environment, `minikube`.
+    `minikube` installation: https://kubernetes.io/docs/setup/learning-environment/minikube/
+    * `kubectl` CLI tool: https://kubernetes.io/docs/tasks/tools/install-kubectl/
+2. Docker and Docker Hub is properly installed and configured on your local system
+    * Installation instruction: https://www.docker.com/get-started
+3. Python (3.6 or above) and required packages: `bentoml`, `fastai`
+    * ```pip install bentoml fastai```
+
+
+## Save pet classification model with BentoML
+
+Follow the notebook from Fastai lesson one to train a pet classification model.
+
+An example base on that notebook is the following:
+
+```python
+from fastai.vision import *
+
+path = untar_data(URLs.PETS)
+path_img = path/'images'
+fnames = get_image_files(path_img)
+pat = re.compile(r'/([^/]+)_\d+.jpg$')
+bs=64
+data = ImageDataBunch.from_name_re(path_img, fnames, pat, ds_tfms=get_transforms(),
+                                   size=299, bs=bs//2).normalize(imagenet_stats)
+learn = create_cnn(data, models.resnet50, metrics=error_rate)
+learn.fit_one_cycle(8)
+learn.unfreeze()
+learn.fit_one_cycle(3, max_lr=slice(1e-6,1e-4))
+```
+
+### Define API server with BentoML
+
+Save the following code to a file named `pet_classification.py`:
```

Comment:
commit this code to the directory? 

can add a line here:

Use BentoML to create an API model server for the pet classification model: